railties/test/generators/app_generator_test.rb

```diff
@@ -872,7 +872,7 @@ def test_spring_binstubs
 
     generator([destination_root], skip_webpack_install: true)
 
-    assert_bundler_command_called("exec spring binstub --all")
+    assert_bundler_command_called("exec spring binstub")
```

Comment:
Is this intentionally removed?

Reply:
Yes, this was actually how I discovered this issue!  The actual `--all` arg was removed in https://github.com/rails/rails/pull/39225/files#diff-5762f42ce6ab7104d2d361e20112979aL434, but this test continued to pass because of the misplaced assertions.